react/src/pages/LoginCognitoBox.js

```diff
@@ -23,8 +23,8 @@ export const LoginCognitoBox = ({ hide }) => {
         onData: (data) => {
             const configuration = {
                 region: data.region,
-                userPoolId: data.user_pool_id,
-                userPoolWebClientId: data.client_id,
+                userPoolId: data.userpool,
```

Comment:
hmm. harder to do that data-driven thing I mentioned above from JS. maybe not worth it.

Reply:
Yeah.